### PR TITLE
[JLINE3-332] Throw IllegalArgumentException in case there is no times…

### PR DESCRIPTION
**Pull request by snuyanzin**
The PR suggests to fix cases mentioned in #332 
it also provides junit test

### CODE REVIEW

File: reader/src/main/java/org/jline/reader/impl/history/DefaultHistory.java [hunk 1]
```diff
@@ -100,12 +101,19 @@ public void load() throws IOException {
...
             if (idx < 0) {
-                throw new IllegalArgumentException("Bad history file syntax! " +
-                        "The history file `" + path + "` may be an older history: " +
-                        "please remove it or use a different history file.");
+                throw new IllegalArgumentException(badHistoryFileSyntax);
             }
-            Instant time = Instant.ofEpochMilli(Long.parseLong(line.substring(0, idx)));
+            Instant time;
+            try {
+              time = Instant.ofEpochMilli(Long.parseLong(line.substring(0, idx)));
```
Comment (gnodet) [inline reader/src/main/java/org/jline/reader/impl/history/DefaultHistory.java]: Could you please change the 2 spaces indentation in a 4 spaces indentation ? That way, I'll merge the PR directly. Thx !

Comment (snuyanzin) [reply reader/src/main/java/org/jline/reader/impl/history/DefaultHistory.java]: Done, please let me know if I need to squash

Comment (gnodet) [reply reader/src/main/java/org/jline/reader/impl/history/DefaultHistory.java]: Yes, please I'd rather have a single commit for that.

Comment (snuyanzin) [reply reader/src/main/java/org/jline/reader/impl/history/DefaultHistory.java]: ok, done

**Pull request merged**